### ISSUE TITLE
Train my self data, the loss is large

### ISSUE THREAD

**Issue by baibaidedongdong** [yzd-v/FGD#65]
When I train my self data, the loss is large, does it normal?The total loss is up 80 in the begining.

![1664189591942](https://user-images.githubusercontent.com/54987492/192259100-4fc357cb-e26b-4d0a-bbfa-b0853f34314a.png)

**Comment by zhaotongdong** [comment]
![1664189700545](https://user-images.githubusercontent.com/54987492/192259343-77f54b11-4bf6-4d39-aa0e-4c28937a84c3.png)

**Comment by yzd-v** [comment]
The original loss with distillation should be much smaller than the original loss when training the student directly.

**Comment by zhaotongdong** [comment]
> The original loss with distillation should be much smaller than the original loss when training the student directly.

Can you share your training loss with me?Thank you!

**Comment by zhaotongdong** [comment]
> The original loss with distillation should be much smaller than the original loss when training the student directly.
![1664518878854](https://user-images.githubusercontent.com/54987492/193203800-a49396d8-706f-43c1-9eee-2559ef4079d6.jpg)
![1664518919196](https://user-images.githubusercontent.com/54987492/193203888-c97fb661-acb5-496a-9a4e-c3e99898fbda.jpg)

你好，我这个是不是有点问题呀，第一张图片是用了fgd，用yoloxl蒸馏yoloxs的损失曲线，第二张图片是单独训练yoloxs的损失曲线，用了fgd之后，yoloxs的损失明显变大了，这个正常吗？

**Comment by zhaotongdong** [comment]
> > The original loss with distillation should be much smaller than the original loss when training the student directly.
> > ![1664518878854](https://user-images.githubusercontent.com/54987492/193203800-a49396d8-706f-43c1-9eee-2559ef4079d6.jpg)
> > ![1664518919196](https://user-images.githubusercontent.com/54987492/193203888-c97fb661-acb5-496a-9a4e-c3e99898fbda.jpg)
> 
> 你好，我这个是不是有点问题呀，第一张图片是用了fgd，用yoloxl蒸馏yoloxs的损失曲线，第二张图片是单独训练yoloxs的损失曲线，用了fgd之后，yoloxs的损失明显变大了，这个正常吗？

这个是在我自己的数据集上的损失曲线

**Comment by yzd-v** [comment]
感觉是不正常的，我找不到yolox的训练log了，你可以按照config跑一下蒸馏，跑一个epoch，和baseline的一个epoch比较一下，应该蒸馏是会明显涨点的
